dashboard.py

```diff
@@ -324,6 +324,9 @@ def make_timeseries(
                     f"{spec.field_name}:Q", title=f"{spec.field_desc} {spec.unit}"
                 ),
                 altair.Tooltip("ci_run_url:N", title="CI Run URL"),
+                altair.Tooltip("cluster_name:N", title="Cluster name"),
+                altair.Tooltip("cluster_id:N", title="Cluster id"),
+                altair.Tooltip("cluster_details_url:N", title="Cluster details"),
```

Comment:
It might also be good to actually change the href attribute above to point to the cluster details page rather than the CI run url.

Reply:
> It might also be good to actually change the href attribute above to point to the cluster details page rather than the CI run url.

What exactly is this href doing here, and where is it visible in the plot?
 I'd like to have a way from the plot to click on the details page but not sure how to achieve this.

Reply:
That's exactly what the href attribute does: when you click on the point you go to the URL. It's currently pointed at the ci workflow run, but it could point to the cluster details page. But perhaps we want to keep it as is. It would be nice to have both, but I'm not sure if that's possible today

Reply:
Oh I see. 
I guess I was wondering how to make the details page available somehow so people can click and go there. But not sure where to expose it.

Reply:
Href is the only place I can think of with the current setup. But we could also design new widgets to display that info.